test/spec/modules/rtbhouseBidAdapter_spec.js

```diff
@@ -132,6 +132,14 @@ describe('RTBHouseAdapter', () => {
       expect(data.source.tid).to.equal('example-transaction-id');
     });
 
+    it('should include bidfloor in request', () => {
```

Comment:
description "... if available" 
We are not making it required, no?

Reply:
Added.